src/examples/table/TableScroll.tsx

```diff
@@ -7,14 +7,19 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import getScrollbarSize from 'dom-helpers/scrollbarSize';
+import { getColor } from '@zendeskgarden/react-theming';
 import { Body, Cell, Head, HeaderCell, HeaderRow, Row, Table } from '@zendeskgarden/react-tables';
 
-const SCROLLBAR_SIZE = getScrollbarSize();
+const StyledHeaderRow = styled(HeaderRow)`
+  border-bottom: none;
```

Comment:
```suggestion
  border-bottom-color: transparent;
```

It's very subtle, but there is undesired compaction that happens when a border is removed on a table with `border-collapse: collapse`.